pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
+    return x / y
```

Comment:
It looks like y should be z
```suggestion
    return x / z
```